packages/xo-server-auth-saml/src/index.js

```diff
@@ -85,7 +85,7 @@ class AuthSamlXoPlugin {
         }
 
         try {
-          done(null, await xo.registerUser('saml', name))
+          done(null, await xo.registerUser2('saml', { user: { id: name, name } }))
```

Comment:
Is the name the best identifier for a SAML user?

Reply:
https://github.com/node-saml/passport-saml/tree/2.x#the-profile-object

Reply:
The `ID` field would be better but it may be `undefined`. Maybe we should also change the plugin config so that the admin can choose what to use?

Reply:
How can this field be optional :confused:

Reply:
I think we should use a fallback chain of fields like `ID ?? name` and make it configurable in the static config?

Reply:
Make what configurable exactly?

Reply:
Never mind, it's too complicated and it tries to answer a problem we don't have, let's use the name for now and we'll regret it later if necessary :stuck_out_tongue: